shell.nix

```diff
@@ -17,7 +17,7 @@ let
   channel = import ./nix/lib/rust.nix { inherit sources; };
   # python environment for tests/bdd
   pytest_inputs = python3.withPackages
-    (ps: with ps; [ virtualenv black ]);
+    (ps: with ps; [ virtualenv grpcio grpcio-tools pytest black ]);
```

Comment:
we're already using pytest, do we need to add it here? @paulyoong

Reply:
I would have thought that we would have hit this before now if we needed it, so I'm guessing we can omit it.